### ISSUE TITLE
can not open file//error opening file

### ISSUE THREAD

**Issue by wenzuolin** [lzx1413/LabelImgTool#10]
I use the LabelImgTool in Win7 with python2.7. However, there is a big question:
ValueError: u'C:/Users/Administrator/Desktop/data/photo/photo1.png' is not in list

**Comment by lzx1413** [comment]
you can try use opendir rather than open image

**Comment by wenzuolin** [comment]
Yeah, when I open the file that I store data when I use the windows system, I will prompt the error openging file, but I will not have this problem with the Linux system.

**Comment by wenzuolin** [comment]
and when i open image , prompt valuerror    my image is not in list
